Field_Simulation/Simulation/WindowsFormsApp1/Form1.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using System.Threading;
+
+namespace WindowsFormsApp1
+{
+    public partial class Form1 : Form
+    {
+        Mutex cleanM = new Mutex();
+        List<int[]> toClean = new List<int[]>();
```

Comment:
`List<List<int>>` or `int[][]` or `int[,]`.

This type declaration... Why? 
If there is no VERY GOOD reason to combine a `List` with `Array`, you shouldn't do that.